src/ameba/rule/style/guard_clause.cr

```diff
@@ -0,0 +1,184 @@
+module Ameba::Rule::Style
+  # Use a guard clause instead of wrapping the code inside a conditional
+  # expression
+  #
+  # ```
+  # # bad
+  # def test
+  #   if something
+  #     work
+  #   end
+  # end
+  #
+  # # good
+  # def test
+  #   return unless something
+  #
+  #   work
+  # end
+  #
+  # # also good
+  # def test
+  #   work if something
+  # end
+  #
+  # # bad
+  # if something
+  #   raise "exception"
+  # else
+  #   ok
+  # end
+  #
+  # # good
+  # raise "exception" if something
+  # ok
+  #
+  # # bad
+  # if something
+  #   foo || raise("exception")
+  # else
+  #   ok
+  # end
+  #
+  # # good
+  # foo || raise("exception") if something
+  # ok
+  # ```
+  #
+  # YAML configuration example:
+  #
+  # ```
+  # Style/GuardClause:
+  #   Enabled: true
+  # ```
+  class GuardClause < Base
+    include AST::Util
+
+    properties do
+      enabled false
```

Comment:
why disabled?

Reply:
I still need to complete the todo below: "check if example is too long for single line"